examples/models/llava/install_requirements.sh

```diff
@@ -13,19 +13,8 @@ pip install --force-reinstall -e examples/third-party/LLaVA
 # not included in the pip install package, but needed in llava
 pip install protobuf
 
-# bitsandbytes depends on numpy 1.x, which is not compatible with numpy 2.x.
-# Reinstall bitsandbytes to make it compatible.
-pip install bitsandbytes -I
-
-# numpy needs to be pin to 1.24. 1.26.4 will error out
-pip install numpy==1.24
-
 # The deps of llava can have different versions than deps of ExecuTorch.
 # For example, torch version required from llava is older than ExecuTorch.
 # To make both work, recover ExecuTorch's original dependencies by rerunning
 # the install_requirements.sh.
 bash -x ./install_requirements.sh --pybind xnnpack
-
-# Newer transformer will give TypeError: LlavaLlamaForCausalLM.forward() got an unexpected keyword argument 'cache_position'
-pip install timm==0.6.13
-pip install transformers==4.38.2
```

Comment:
This is removed because it's already in root level install_requirements.sh?

Reply:
The root script is installing the latest versions. To me when I tried exporting llava locally, the latest HF versions actually work well. Unless there are specific reasons to hold us back to these old versions, we should use latest versions.

Reply:
Can you fix `[test-export-llava-linux /](https://github.com/pytorch/executorch/actions/runs/10015173739/job/27686153177?pr=4320#logs)`

Reply:
> Can you fix [test-export-llava-linux /](https://github.com/pytorch/executorch/actions/runs/10015173739/job/27686153177?pr=4320#logs)

I haven't got a chance to look into the failure. But I notice that the issue is thrown from torchao 0.1. @jerryzh168 Is this something you may know? Also we are still pinning torchao 0.1 even in main, should we bump it up to newer version? What would be the version we want to use for the upcoming releases including Beta release?

Reply:
Locally I can export the llava with xnnpack w/o a problem w/ torch 2.5.0.dev20240716 + torchao 0.1

Reply:
We are not supporting ao v0.1, please use the latest stable release with 0.3.1

This error is strange and I answered here https://github.com/pytorch/ao/issues/535#issuecomment-2246223871 it's strange that triton is even installed on this machine if the expectation is that this is a cpu only job

torchao 0.3.1 has no dependencies so it shouldnt even be installing a torch version

Reply:
@msaroufim It turns out that the third-party llava (very old code) pulled triton and installed 2.1.1, which doesn't work with torchao.

Reply:
> We are not supporting ao v0.1, please use the latest stable release with 0.3.1

@msaroufim Prefer to address one issue a time. We can update torchao dep to 0.3.1. 

torchao is not the only package has this issue. That's why I have to upgrade HF packages and resolve all the deps problems manually. I think @dbort has a proposal to use ">=" for deps in general to make sure our packages will not stuck at some stale versions.